PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+<!-- Hello,
+
+Thank you for submitting a pull request, that's awesome! Please add some info (if applicable) to give us some context on the PR.
+
+Things to consider before you submit the PR:
+
+* [x] Has `CHANGELOG.md` been updated?
+* [] Are tests passing locally?
+* [] Are the files formatted correctly?
+* [] Did you add unit tests?
+
+We will review the PR as soon as possible, leave feedback, add a tag, etc. and let you know what's going on.
+
+Cheers!
+
+The App Center team -->
+
+Things to consider before you submit the PR:
+
+* [x] Has `CHANGELOG.md` been updated?
+* [] Are tests passing locally?
```

Comment:
Every PR will have tasks associated to them if you use check boxes, is that intended?